todo_app/templates/index.html

```diff
@@ -18,15 +18,15 @@ <h2>Things to do</h2>
               <td scope="col">Move to done?</td>
             </tr>
         </thead>
-        {{ items.name }}
-        {% for card in items %}
+        {{ view_model.cards.name }}
+        {% for card in view_model.cards %}
```

Comment:
Worth noting that our assumption is that we'd switch to using `.to_do_cards` and `.done_cards` directly, but that doesn't always match people's preferred UX - absolutely fine to leave this as is, as long as we consciously acknowledge this means our unit tests are testing something we never use, so they're functionally useless outside of learning (but then that's the key point!)

Reply:
Sorry, I don't understand what you mean here?

Reply:
Ah sorry; the key bit is that we've added three new properties (`to_do_cards`, `doing_cards`, `done_cards`) but we aren't actually using those anywhere, even though they're also our only tests, which feels a little pointless!

The expectation was that we'd use those in the template, e.g. like:
```jinja
        {% for card in view_model.to_do_cards %}
            <tr>
                <td>{{ card.name }}</td>
                <td>{{ card.status }}</td>
                <td>
                    <form id="complete-card" action="/complete-card" method="post">           
                      <input type="hidden" name="card_id" id="card_id" value="{{card.id}}" class="hidden" />
                      <input type="submit" value="Done" class="btn btn-primary" />
                    </form>
                </td>
            </tr>
        {% endfor %}
        {% for card in view_model.done_cards %}
            <tr>
                <td>{{ card.name }}</td>
                <td>{{ card.status }}</td>
                <td>
                </td>
            </tr>
        {% endfor %}

```

But I know that doesn't always match how people prefer their code/UX, so fine to ignore here but worth being conscious that creating those properties was a very artificial scenario in that case!